tests/domain/test_request.py

```diff
@@ -1,27 +1,12 @@
 import pytest
 
 from falco import Request
-from output_pb2 import request
+from outputs_pb2 import request
 
 
 class TestRequest:
```

Comment:
I think this test doesn't have a purpose anymore

Reply:
Yep, this doesn't have a purpose anymore.

Anyway, I'd leave it here for now for when we'll have fields into the request messages (eg., rule/alerts tags)